api/src/test/java/org/pmiops/workbench/actionaudit/ActionAuditServiceTest.kt

```diff
@@ -85,67 +80,74 @@ class ActionAuditServiceTest {
+            // Logging passes numeric json fields as doubles when building a JsonPayload
+            assertThat(payloadMap[AuditColumn.AGENT_ID.name]).isEqualTo(AGENT_ID_1.toDouble())
+        }
     }
 
     @Test
     fun testSendsExpectedColumnNames() {
         actionAuditService!!.send(event1!!)
-        verify<Logging>(mockLogging).write(logEntryListCaptor!!.capture())
-        val entryList = logEntryListCaptor.value
-        assertThat(entryList.size).isEqualTo(1)
-        val entry = entryList[0]
-        val jsonPayload = entry.getPayload<JsonPayload>()
-
-        for (key in jsonPayload.dataAsMap.keys) {
-            assertThat(Arrays.stream(AuditColumn.values()).anyMatch { col -> col.toString() == key })
-                    .isTrue()
+        argumentCaptor<List<LogEntry>>().apply {
+            verify(mockLogging).write(capture())
+            val entryList = firstValue
+            assertThat(entryList.size).isEqualTo(1)
+            val entry = entryList[0]
+            val jsonPayload = entry.getPayload<JsonPayload>()
+
+            for (key in jsonPayload.dataAsMap.keys) {
+                assertThat(Arrays.stream(AuditColumn.values()).anyMatch { col -> col.toString() == key })
+                        .isTrue()
+            }
         }
     }
 
     @Test
     fun testSendsMultipleEventsAsSingleAction() {
         actionAuditService!!.send(ImmutableList.of(event1!!, event2!!))
-        verify<Logging>(mockLogging).write(logEntryListCaptor!!.capture())
-        val entryList = logEntryListCaptor.value
-        assertThat(entryList.size).isEqualTo(2)
-
-        val payloads = entryList
-                .map { it.getPayload<JsonPayload>() }
-
-        assertThat(
-                payloads
-                        .map { it.getDataAsMap() }
-                        .map { it[AuditColumn.ACTION_ID.name] }
-                        .distinct()
-                        .count())
-                .isEqualTo(1)
+        argumentCaptor<List<LogEntry>>().apply {
+            verify(mockLogging).write(capture())
+            val entryList = firstValue
+            assertThat(entryList.size).isEqualTo(2)
+
+            val payloads = entryList
+                    .map { it.getPayload<JsonPayload>() }
+
+            assertThat(
+                    payloads
+                            .map { it.dataAsMap }
+                            .map { it[AuditColumn.ACTION_ID.name] }
+                            .distinct()
+                            .count())
+                    .isEqualTo(1)
+        }
     }
 
     @Test
     fun testSendWithEmptyCollectionDoesNotCallCloudLoggingApi() {
         actionAuditService!!.send(emptySet())
-        verify<Logging>(mockLogging, never()).write(anyList())
+        argumentCaptor<List<LogEntry>>().apply {
+            verify(mockLogging, never()).write(any())
+        }
     }
 
     companion object {
-
         private const val AGENT_ID_1 = 101L
```

Comment:
Eric brought this up too, and I figured we should wait on pure aesthetical decisions until we pick a style guide, and then we could either adjust the linter(s) or do a batch switch.

I think I kept them at the bottom because that's where IntelliJ put them on translation. Looking at [this style guide](https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout), it seems they do it that way too (though I only just now searched for it).